sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py

```diff
@@ -109,13 +109,11 @@ class CertificateClient(AsyncKeyVaultClientBase):
         return await async_poller(command, create_certificate_operation, None, create_certificate_polling)
 
     @distributed_trace_async
-    async def get_certificate(self, certificate_name: str, **kwargs: "**Any") -> KeyVaultCertificate:
-        """Gets a certificate with its management policy attached.
+    async def get_certificate(self, certificate_name: str, **kwargs: "Any") -> KeyVaultCertificate:
+        """Gets a certificate with its management policy attached. Requires certificates/get permission.
 
-
-        This operation requires the certificates/get permission. Does not accept the
-        version of the certificate as a parameter. If you wish to specify version, use
-        the get_certificate_version function and specify the desired version.
+        Does not accept the version of the certificate as a parameter. If you wish to specify version,
```

Comment:
Suggestion:  ~If you wish to specify version~ To get a specific version of the certificate

Reply:
sounds better, thanks!